src/collections/configure/index.ts

```diff
@@ -117,6 +119,26 @@ const configure = {
           : undefined,
     };
   },
+  objectTTL: (options?: {
+    enabled?: boolean;
```

Comment:
I also would not add enabled as an option here. If you set TTL for creation it is enabled